src/test/java/org/folio/rest/impl/PostGobiOrdersHelperTest.java

```diff
@@ -375,5 +412,145 @@ public final void testLookupOrderMappings(TestContext context) throws Exception
         });
     });
   }
+
+  @Test
+  public final void testLookupWorkflowStatusId(TestContext context) throws Exception {
+    final Async async = context.async();
+    final Vertx vertx = Vertx.vertx();
+    final HttpServer server = vertx.createHttpServer();
```

Comment:
Why not use the mock Server?

Reply:
No reason not to. I was rushing to get some unit tests in on Friday and didn't think of it. We could/should definitely refactor these tests anyway. There is no need to have server startup in each test case. I'll work with Varun or Kruthi to get these unit tests better sorted out.